src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java

```diff
@@ -65,31 +70,37 @@ public WebACRecipesIT() {
 
     @Before
     public void setUp() throws ClientProtocolException, IOException {
-        // TODO: this should be factored out into a repeatable method to set up each scenario
-        final HttpPost postRequest = new HttpPost(serverAddress);
-
-        final String creds = "username:password";
-        // in test configuration we don't need real passwords
-        final String encCreds = new String(Base64.encodeBase64(creds.getBytes()));
-        final String basic = "Basic " + encCreds;
-        postRequest.setHeader("Authorization", basic);
-
-        final InputStream acl = this.getClass().getResourceAsStream("/acls/01/acl.ttl");
-        final InputStreamEntity acl_entity = new InputStreamEntity(acl);
-        postRequest.setEntity(acl_entity);
-        postRequest.setHeader("Content-Type", "text/turtle;charset=UTF-8");
-
-        // XXX: this is currently failing in the test repository with a
-        // "java.lang.VerifyError: Bad type on operand stack"
-        // see https://gist.github.com/peichman-umd/7f2eb8833ef8cd0cdfc1#gistcomment-1566271
-        final HttpResponse res = client.execute(postRequest);
-        System.err.println(res.getStatusLine());
-
         logger.debug("setup complete");
     }
 
     @Test
-    public void test() {
+    public void scenario1() throws Exception {
+        logger.info("Running scenario1");
+        final String objA = getRandomPid();
+        final HttpPut method = super.putObjMethod(serverAddress + "/" + objA);
+        final FileEntity acl =
+                new FileEntity(new File(classLoader.getResource("acls/01/acl.ttl").getFile()));
+        setAuth(method, "fedoraAdmin");
+        method.setHeader("Content-type", "text/turtle");
+        method.setEntity(acl);
+        try (final CloseableHttpResponse response = super.execute(method)) {
+            final String content = EntityUtils.toString(response.getEntity());
```

Comment:
The body of the response does not help much, and in the name of less/cleaner code, I would suggest removing it and changing the assertion below to:
```bash
assertEquals(CREATED.getStatusCode(), status);
```